merlin/systems/dag/ops/session_filter.py

```diff
@@ -48,6 +79,33 @@ def compute_input_schema(
         deps_schema: Schema,
         selector: ColumnSelector,
     ) -> Schema:
+        """
+        Compute the input schema of this node given the root, parents and dependencies schemas of
```

Comment:
serial comma, please: "...root, parents, and dependencies..."

Ending punctuation in the foll descriptions, please.